rails-5/Dockerfile

```diff
@@ -1,14 +1,31 @@
-FROM ruby:2.3-alpine
+FROM avvo/alpine:3.5
 MAINTAINER Greg Orlov <greg@avvo.com>
 
 RUN apk update && \
     apk upgrade && \
-    apk add zlib-dev libxml2-dev libxslt-dev tzdata build-base linux-headers openssl-dev ca-certificates && \
-    apk add ruby-dev ruby-io-console ruby-json ruby-bigdecimal && \
-    update-ca-certificates && \
+    apk add \
+      'ruby<2.4' \
+      ruby-dev \
+      ruby-libs \
+      ruby-minitest \
+      ruby-rdoc \
+      ruby-bigdecimal \
+      ruby-io-console \
+      ruby-json \
+      ruby-irb \
+      ruby-rake && \
+    apk add zlib-dev \
```

Comment:
I'd get rid of this superfluous `apk add` then alpha-order all the package names